src/main/java/hudson/plugins/git/util/GitUtils.java

```diff
@@ -272,6 +272,15 @@ private static void addEnvironmentContributingActionsValues(EnvVars env, Abstrac
                 }
             }
         }
+        
+        // Use the default parameter values (if any) instead of the ones from the last build
```

Comment:
What do you mean ?
This is only for the polling, and the git plugin does currently provide such values. The difference is that it currently uses the last used value instead of the configured default value.